tests/shared/src/test/scala/scala/meta/tests/parsers/Scala213Suite.scala

```diff
@@ -128,6 +128,38 @@ class Scala213Suite extends ParseSuite {
     )(dialects.Scala213Source3)
   }
 
+  test("issue-2880") {
+    runAssert(
+      """|Flow {
+         |    b.add()
+         |    input_< ~> filtering ~> removeItems.in0
```

Comment:
which part here was "multiple infix operators"? `input_<` and then `~>`?

Reply:
Yes, that was it though I don't neccessarily understand the exact rule: https://docs.scala-lang.org/scala3/reference/changed-features/operators.html#syntax-change

There is a test failing in dotty that I should take a look at.

Reply:
I think the rule is actually different, need to rework this.